src/tox_current_env/hooks.py

```diff
@@ -136,7 +148,10 @@ def tox_runtest(venv, redirect):
     """If --print-deps-only, prints deps instead of running tests"""
     config = venv.envconfig.config
     unsupported_raise(config, venv)
+    if config.option.print_deps_path is not None:
+        with open(config.option.print_deps_path, "a", encoding="utf-8") as f:
+            print(*venv.get_resolved_dependencies(), sep="\n", file=f)
+        return True
```

Comment:
The `return True` means `--print-deps-only` won't have an effect with `--print-deps-to-file`.
Should these options be mutually exclusive?

Reply:
They should.

Reply:
Done. Please re-review (added in extra commit, so it should be easier).